stable/rethinkdb/templates/rethinkdb-cluster-stateful-set.yaml

```diff
@@ -101,6 +98,9 @@ spec:
       resources:
         requests:
           storage: {{ .Values.cluster.persistentVolume.size | quote }}
+    {{- if .Values.cluster.storageClass.enabled }}
```

Comment:
@gyliu513 Curious why we didn't check `eq '-'` here like the others? (note, I opened a wider discussion on this for future here https://github.com/Masterminds/sprig/issues/53, even though it shouldn't impact these PRs being merged).

Reply:
@scottrigby You may see that this chart is defining `storageClass` by itself here https://github.com/kubernetes/charts/blob/master/stable/rethinkdb/templates/rethinkdb-cluster-storage-class.yaml , so we can use the `storageClass` only when `.Values.cluster.storageClass.enabled`, otherwise, we can use default `storageClass`, this will keep same logic as before.